sigmf/sigmffile.py

```diff
@@ -890,14 +944,23 @@ def get_dataset_filename_from_metadata(meta_fn, metadata=None):
     return None
 
 
-def fromarchive(archive_path, dir=None):
-    """Extract an archive and return a SigMFFile.
+def fromarchive(archive_path):
```

Comment:
I would think it would be better (as it's less likely to break someone's existing code that specifies `dir`) to leave it as an argument, even if it's not used.